src/pages/User/Repository.vue

```diff
@@ -161,18 +161,33 @@ export default {
 
   created() {
     this.$store.dispatch('GetAllUsers');
-    this.$store.dispatch('GetRepositoryNote', this.$route.params.id)
     this.$store.dispatch('GetRepositoryTags', this.$route.params.id)
+    this.loadComment();
   },
 
   methods: {
+    loadComment() {
+      this.$store.dispatch('GetRepositoryNote', this.$route.params.id);
+    },
     onClickComment() {
       this.$store.dispatch('AddRepositoryNote', {
         id: this.repository.id,
         from: this.repository.user.uuid,
         text: this.form.comment,
+      })
+      .then((res)=>{
+        this.loadComment();
```

Comment:
Based on the design, there is no need to run `loadComment`.

If you look at `module/repository.js` Line 128.

```
AddRepositoryNote({ rootState }, data) {
      data.from = rootState.user.userId
      console.log(data)
      return new Promise((resolve, reject) => {
        this.$repository.repository.createNote(data.id, data.from, data.to, data.text)
          .then(res => {
            Note.insert(res.data)
            resolve(res.data)
          })
          .catch(err => {
            reject(err)
          })
      })
    }
```

Every time it has successfully create a comment, it would insert the data to the vuex database. So, its a redundant to call `loadComment` back.